### ISSUE TITLE
Does this library work with a lists that has section headings?

### ISSUE THREAD

**Issue by JeremyColton** [bvaughn/react-window#559]
Hi, my list is separated into sections. Each section has a heading and then numerous rows of items. Eg

```
<div class="section">
   <h6>SECTION 1</h6>
   <div class="section-contents">
      <div class="row">
         <div class="float-left">Item 1</div>
         <div class="float-right">Item 2</div>
      </div>
      <div class="row">
         <div class="float-left">Item 1</div>
         <div class="float-right">Item 2</div>
      </div>
.
.
.
   </div>
</div>
```
Can your library work with such a list?

Thanks

**Comment by Mikoz93** [comment]
@JeremyColton Not really, I mean it will work fine in the sense that it will virtualize items as expected, but if you'll need to use 'scrollTo' or 'scrollToItem' methods then you might run into unexpected behavior because of the extra items.

**Comment by slavab89** [comment]
We needed that feature as well and we didnt quite make it work.
It depends on the height of each section - the amount of items in each section. For us, every section had a different amount of items under it, and some were quite long so it means you have to calculate/set the height of each section individually.
Moreover, if each section is defined as an item, it means that it will get rendered into and out of the DOM upon "virtualization" and if the sections are "heavy" then you'll notice the prformance impact.

What we ended up doing (For now) is to have sections and items as regular list rows. The array of list rows contains both "section" items, and "regular" items. This makes the scrolling and rendering work better as each row is rendered on its own.

We are still working on improving this solution because, as said before, our items are highly dynamic in their height and because of that, jumping to some sections/items is difficult because we dont always know what is the exact height of the item at the end of the list (Since it was never rendered)
